collections/copernicus-dem.yaml

```diff
@@ -21,6 +21,7 @@ Contact: https://forum.sentinel-hub.com/c/euro-data-cube/25
 Provider: "[Copernicus](https://copernicus.eu/)"
 ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
 Tags:
+  - copernicus services
```

Comment:
Are you sure this is part of copernicus services? It was not in Grega's list and I could not find a explicit source saying this. If you are sure feel free to resolve and merge.

Reply:
Wasn't sure about that one, I'll leave it out.

Reply:
Does it still need the `RegistryEntryLastModified` change after that?